### ISSUE TITLE
tflite vs ONNXRuntime accuracy

### ISSUE THREAD

**Issue by adityat77** [microsoft/onnxruntime#12924]
### Describe the issue

Given a benchmarked tflite model ( taking mobilenetv2 as an example), I tried converting it to ONNX using tf2onnx converter. Conversion wise the model seems to be fine. But running a simple forward pass on both tflite and onnx models, is giving a **high MSE in the output layer**. Am I doing something wrong while conversion / comparison. Attaching model and steps i followed to convert and comparison script. 

tflite model: [**mobilenet_v2**](https://github.com/google-coral/edgetpu/blob/master/test_data/mobilenet_v2_1.0_224_quant.tflite)

convert to onnx: `python3 -m tf2onnx.convert --opset 13 --tflite mobilenet_v2.tflite --output q_mobilenet_v2.onnx`

### To reproduce

comparison script: 

```
#!/bin/env python3
import onnx, tensorflow as tf, numpy as np, onnxruntime

TFLITE_FILE_PATH = 'mobilenet_v2.tflite'
ONNX_FILE_PATH = 'q_mobilenet_v2.onnx'

if __name__=='__main__':
    interpreter = tf.lite.Interpreter(TFLITE_FILE_PATH)
    interpreter.allocate_tensors()
    input_details = interpreter.get_input_details()
    output_details = interpreter.get_output_details()
    input_shape = input_details[0]['shape']

    sess_options = onnxruntime.SessionOptions()
    sess_options.graph_optimization_level = onnxruntime.GraphOptimizationLevel.ORT_DISABLE_ALL
    session = onnxruntime.InferenceSession(ONNX_FILE_PATH, sess_options)
    output_name = [n.name for n in session.get_outputs()]
    input_node_name = [n.name for n in session.get_inputs()]
    
    # compare on 10 random data 
    for i in range(10):
        input_data = np.ones(shape=(input_shape), dtype=np.uint8)#np.array(np.random.randint(0,255,size=input_shape, dtype=np.uint8))
        print(input_details[0]['index'])
        interpreter.set_tensor(input_details[0]['index'], input_data)
        interpreter.invoke()
        o_tfl = interpreter.get_tensor(output_details[0]['index'])
        o_onnx = session.run(output_name, {'input':input_data})[0]
        
        diff = abs(o_tfl - o_onnx)
        MSE = np.square(diff).mean()
        E_AMAX = diff.max()
        E_PAMX = E_AMAX/np.abs(o_onnx).mean()
        print(f"output_tfl: {o_tfl}\noutput_onnx:{o_onnx}\nMSE:    {MSE}\nE_AMAX: {E_AMAX}\nE_PAMX: {E_PAMX}\n")
        print('___________________________\n')
```

### Urgency

I'm looking at accuracy of a QAT pytorch model -> ONNX and how much accuracy will I be able to retain. 

### Platform

Linux

### OS Version

4.18.0

### ONNX Runtime Installation

Released Package

### ONNX Runtime Version or Commit ID

1.12.1

### ONNX Runtime API

Python

### Architecture

X86

### Execution Provider

Default CPU

**Comment by wangyems** [comment]
Can you please open an issue here: https://github.com/onnx/tensorflow-onnx/issues

**Comment by adityat77** [comment]
sure.

**Comment by faxu** [comment]
Closing here. Tracked in: https://github.com/onnx/tensorflow-onnx/issues/2041
